src/styles/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-items: center;
+  width: auto;
+  justify-content: space-between;
+  line-height: 16px;
+  padding: 16px 20px 0 16px;
+
+  &__star {
+    background-size: cover;
+    width: 12px;
+    height: 12px;
+    margin-left: 4px;
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__star--inactive {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
use your solution from a previous `stars` task where you worked with stars `modifiers`
```suggestion
```